pkg/core/error_handler.go

```
@@ -46,24 +46,23 @@ type FieldEncodingErrorManager func()
 
 	return func() {
 		maybeEncodingError := optional.FlatMap(maybeFallbackField, h.encodeAndLift)
 		maybeFallbackFailed := optional.Map(maybeEncodingError, func(_ error) Field {
-			return String(field.Name(), h.cfg.FallbackErrorMessage)
+			return String(field.Name(), h.cfg.FallbackErrorMessage())
 		})
 		optional.FlatMap(maybeFallbackFailed, h.encodeAndLift)
 	}
 }
 
+func (h *ErrorHandler) doNothing() {
```

golangci-lint (2.5.0)

[error] unexported method "doNothing" for struct "ErrorHandler" should be placed after the exported method "AggregatedError"

(funcorder)